src/components/new-feature.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { Link } from 'gatsby'
+
+// import "./footer.scss"
+import ArrowRightAlt from '@material-ui/icons/ArrowRightAlt';
+
+
+const NewFeature = (props) => {
+
+    return (
+        <div className="mt-4">
+            <h3>{props.heading}</h3>
+            <div>{props.date}</div>
+            <img src={props.image} alt={props.image_titles} className="mw-100 pt-3"/>
+            <div className="pt-3">{props.description}</div>
+            <div className="pt-3"><Link to={props.pagePath}>See it in action <ArrowRightAlt/></Link></div>
```

Comment:
Should "See it in action" be pulled from Contentful?

Reply:
That would be nice so we can change it when the language changes. Kind of a pain to get it in there though. :/ I guess we could create a new WhatsNewPageContent singleton type?

We could also load the names of the months in there.

Reply:
If there is no value in `props.pagePath`, could we not display the link?

Reply:
Good catch! I just added a new field for the "link name" called `pagePathLinkName`...might not be the best for now.
I think we can revisit with what John suggest later as I'm not 100% sure on what it means.

Reply:
sounds good. in that case, maybe we can just look into using some built-in library to get the month names and then we won't have to load them into Contentful?